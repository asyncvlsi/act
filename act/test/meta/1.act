template<preal w>
defproc foo(bool a, b)
{
  prs {
    a <2.5*w> -> b-
   ~a <1.3*w> -> b+
  }
}


foo<4.0> f;
