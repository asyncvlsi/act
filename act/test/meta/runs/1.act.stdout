 defproc foo<4>(bool a; bool b)
{
/* instances */
prs {
a<10> -> b-
~a<5.2> -> b+
}
}
template <preal w> defproc foo(bool a; bool b)
{
/* instances */
prs {
a<2.5*w> -> b-
~a<1.3*w> -> b+
}
}
/* instances */
foo<4> f;
